docs/MSPX.md

```diff
@@ -24,28 +26,87 @@ MspX is enabled by this setting in the receiver:
 
+***Notes***:
+RC Link Statistics are only send to the Flight controller via MSP, if "rc override" or "rc channels" is enabled. 
+MSP-RC messages will always override other RC Link inputs. If CRSF is connected and CRSF selected as the RC Protocol in INAV, MSP-RC still has priority if enabled in mLRS and connected to a MSP Enabled UART.
+
 ## INAV Configuration
 
-The baudrate should be set to 115200 as lower baudrates have shown irregular telemetry update rates on the radio due to the lack of flow control on the INAV side. 
+The baudrate should be set to 115200 or higher for consistent dataflow and RC control. Baudrates of less than 57600 can cause message loss on the INAV side and result in a inconsistent RC control at 50Hz RC Rate, if MSP-RC is used (INAV Limitation). 
 
-INAV 8.0 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8.0 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
+INAV 8 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
 
 To use a mLRS receiver with INAV in MspX mode, the following settings have to be applied in INAV:
 - Enable MSP for the Serial Port the mLRS receiver is connected to (UART 2 is recommended on most STM32 flight controllers; do not enable "Serial RX").
 - Set the baudrate to 115200 or higher.
   
 <img src="images/MSPX_ports.png" width="720px">
 
-- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP, and save settings.
+- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP and also set the RSSI Source to MSP (INAV 8 and later) and save settings.
   
-<img src="images/MSPX_receivermode.png" width="720px">
+<img src="images/MSPX_receiver.png" width="720px">
 
-- When connecting INAV 8.0 Configurator through mLRS it is highly recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions earlier than 8.0, such as INAV 7.1).
+- When connecting INAV 8 Configurator through mLRS for flight monitoring, it is recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions earlier than 8, such as INAV 7.1).
 
 <img src="images/MSPX_wirelessmode.png" width="360px">
 
 If your radio is connected, you should now be able to see the channel values update when you move the radio sticks. No further settings are needed and telemetry will work for EdgeTX/OpenTX radios after scanning for sensors.
+
+## Connection Types with different INAV Versions
```

Comment:
Oh wow, I didn't see this before :) I suppose I would prefer to have the INAV 8 bit first.  Additionally, I think the MSP-RC 37 Hz limitation should be mentioned earlier in the MSP-RC stuff above.  If I read correctly there is otherwise no limitation to INAV 8?

Reply:
Good call. I added that to the Notes in the intro as well and also put the INAV 8 overview first. 
No other limitations in INAV 8 as far as we know. the apparently inconsistent RC rate is not as bad as it looks in the logs and is exaggerated by aliasing. Its impossible to notice and I tested that in flight as well.